sql/calendar.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS calendar_logs;
+DROP TABLE IF EXISTS calendar_events;
+CREATE TABLE calendar_logs(
+    log_id                 INT          NOT NULL AUTO_INCREMENT,
+    username               VARCHAR(32)  NOT NULL, 
+    calendar_id            VARCHAR(25)  NOT NULL, 
+    calendar_gmail         VARCHAR(50)  NOT NULL,
+    acl_id                 VARCHAR(50)  NOT NULL,
+    request_time           DATETIME     NOT NULL, 
+    request_permission     CHAR(4)     NOT NULL, 
+    PRIMARY KEY (log_id),
+);
+
+CREATE TABLE calendar_events(
+    event_id                 INT          NOT NULL AUTO_INCREMENT,
+    username               VARCHAR(32) , -- Who created this even
```

Comment:
Might want to clarify that this is not a Donut username

Reply:
It is, although I was debating on if i should log it.

Reply:
Hmm, if it's a Donut user, then I would store `user_id` instead